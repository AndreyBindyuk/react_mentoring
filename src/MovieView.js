import React from 'react';
import MovieSingleView from "./movieItemImpl/MovieSingleView"
import MovieItemList from './movieItemImpl/MovieItemList';
import Footer  from './frameImpl/Footer';

class MovieView extends React.Component{
    render(){
        return(
            <div>
            <MovieSingleView />
            <MovieItemList />
            <Footer />
            </div>
        );
    }
}

export default MovieView;