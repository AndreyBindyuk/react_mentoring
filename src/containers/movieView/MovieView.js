import React from 'react';
import MovieSingleView from "../movieSingleView/MovieSingleView"
import MovieItemList from '../movieList/MovieItemList';
import Footer  from '../../shared/footer/Footer';

class MovieView extends React.Component{
    render(){
        return(
            <div>
            <MovieSingleView />
            <MovieItemList />
            <Footer />
            </div>
        );
    }
}

export default MovieView;