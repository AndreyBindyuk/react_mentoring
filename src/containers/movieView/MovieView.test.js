import React from 'react';
import { shallow } from 'enzyme';
import MovieView from './MovieView';

describe('<MovieView />', () => {
  it('render internal components correctly', () => {
    const ModalTriggerComponent = shallow(<MovieView />);
    expect(ModalTriggerComponent).toMatchSnapshot();
  });
});
