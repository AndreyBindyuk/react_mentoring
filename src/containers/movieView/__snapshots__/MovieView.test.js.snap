// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieView /> render internal components correctly 1`] = `ShallowWrapper {}`;
